<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPropertyTemplateDialog</class>
 <widget class="QDialog" name="AddPropertyTemplateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Common Property</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leName"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="gbScope">
     <property name="title">
      <string>Scope</string>
     </property>
     <layout class="QGridLayout" name="gridLayout"/>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="comboTypes"/>
     </item>
    </layout>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="teValues">
     <property name="placeholderText">
      <string>Comma separated values</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>14</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Validation pattern:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="leValidationPattern"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Serialization type</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="rbAttribute">
        <property name="text">
         <string>Attribute</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbProperty">
        <property name="text">
         <string>Property</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddPropertyTemplateDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddPropertyTemplateDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
